### PR TITLE
Persistence options

### PR DESCRIPTION
**Pull request by cabelo**
Saving the disk image for use in other processes or software

### CODE REVIEW

**Comment by stefanhaustein** [comment]
Thanks, but why not just use output redirection for this case?

**Comment by cabelo** [comment]
> Thanks, but why not just use output redirection for this case?

Not work ... See:

https://imgur.com/y1i0GK4

**Comment by stefanhaustein** [comment]
Didn't somebody mention this would be because the queried terminal width would be 0 in this case? If that's the reason, wouldn't it be cleaner to just assume a more reasonable width/height if the queried value is 0?

**Comment by cabelo** [comment]
Sorry you're right, when I used the constant value 0 it worked.
How do you feel about leaving like this?

I am compiling and packaging for other distributions:

https://software.opensuse.org//download.html?project=home%3Acabelo%3Aheroes&package=tiv

File: src/main/cpp/tiv.cpp [hunk 1]
```diff
@@ -401,8 +401,11 @@ enum Mode {AUTO, THUMBNAILS, FULL_SIZE};
 
 int main(int argc, char* argv[]) {
   struct winsize w;
+  ioctl(STDOUT_FILENO, TIOCGWINSZ, &w);
+
+  if(w.ws_col == 0 && w.ws_col == 0)
```
Comment (stefanhaustein) [inline src/main/cpp/tiv.cpp]: Cool, thanks a lot!

Can you add curlies around the conditional statement to make it more obvious? O:)

Comment (cabelo) [reply src/main/cpp/tiv.cpp]:
```suggestion
  if(w.ws_col == 0 && w.ws_col == 0)
```
Sorry,  I did not understand "add curlies around the conditional"

Comment (cabelo) [reply src/main/cpp/tiv.cpp]: done

Comment (stefanhaustein) [inline src/main/cpp/tiv.cpp]:
```
if (w.ws_col == 0 && w.ws_col == 0) {
   ioctl(0, TIOCGWINSZ, &w);
}
```

Comment (cabelo) [reply src/main/cpp/tiv.cpp]: Ok... I understand now ...

Comment (cabelo) [reply src/main/cpp/tiv.cpp]: done

**Comment by stefanhaustein** [comment]
Hey sorry for the confusion, I didn't see the full diff and thought the change was limited to the 0 problem now. Wouldn't solving the 0 problem solve the redirect option without adding a new -o parameter? If redirection works, the -o parameter seems redundant?

**Comment by cabelo** [comment]
I'll create a resquest just by checking the cols and rows ok?

**Pull request merged**